src/main/java/org/tron/common/runtime/vm/program/Storage.java

```diff
@@ -21,6 +25,15 @@ public Storage(byte[] address, StorageRowStore store) {
     this.store = store;
   }
 
+  public Storage(Storage storage) {
+    this.addrHash = storage.addrHash.clone();
+    this.store = storage.store;
+    storage.getRowCache().forEach((rowKey, row) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Specify a type for: 'rowKey', 'row' ([squid:S2211](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2211))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1667&issues=AWbODGo-LpqqSUlcKsP9&open=AWbODGo-LpqqSUlcKsP9)

[ISSUE_KEY]: AWbODGo-LpqqSUlcKsP9